client/app/components/faq/faqlist.js

```diff
@@ -35,94 +30,97 @@ class FaqList extends Component {
 
 	componentDidMount() {
 		this.fetchData()
+			// eslint-disable-next-line no-console
 			.then(() => console.log("Finished mounting!"))
+			// eslint-disable-next-line no-console
 			.catch(err => console.error(err));
 	}
 
 	componentWillReceiveProps(nextProps) {
 		// Avoiding refresh if possible
-	  if (nextProps.currentQuestions !== this.state.currentQuestions) {
-		 this.setState({ currentQuestions: nextProps.currentQuestions });
-	  }
+		if (nextProps.currentQuestions !== this.state.currentQuestions) {
+			this.setState({ currentQuestions: nextProps.currentQuestions });
+		}
 	}
 
-	render() {
-		return (
-			<ScrollView
-				style={FaqStyles.faqPadding, {backgroundColor: 'white'}}
-				refreshControl={
-					<RefreshControl
-						refreshing={this.state.isRefreshing}
-						onRefresh={this._onRefresh.bind(this)}
-					/>
-				}
-			>
-				<FlatList
-					data={this.state.currentQuestions}
-					renderItem={this._renderListItem}
-					extraData={this.state}
-					keyExtractor={this._keyExtractor}
-					ListHeaderComponent={this._renderHeader}
-				/>
-			</ScrollView>
-		);
+	onRefresh = () => {
+		this.setState({isRefreshing: true});
+		this.fetchData().then(() => {
+			this.setState({isRefreshing: false});
+		});
 	}
 
 	fetchData() {
 		return fetch(`${API_URL}/list`)
 			.then(response => response.json())
 			.then(faqList => this.props.onFAQLoaded(faqList))
+			// eslint-disable-next-line no-console
 			.catch(err => console.error(err));
 	}
 
-	_onRefresh() {
-		this.setState({isRefreshing: true});
-		this.fetchData().then(() => {
-			this.setState({isRefreshing: false});
-		});
-	}
-
-	_renderListItem = ({ item, index }) => {
+	renderListItem({ item, index }) {
 		let rowBg = index % 2 == 1 ? FaqStyles.faqListItemOdd : null;
+		
+		const icon = (
+			<Icon
+				name="chevron-thin-right"
+				size={30}
+				color="#787878"
+				style={{marginTop: 5}}
+			/>
+		);
+
 		return (
 			<ListItem
-				containerStyle={FaqStyles.faqListItem, rowBg}
+				containerStyle={{...FaqStyles.faqListItem, rowBg}}
```

Comment:
Temporarily just using the [object spread operator](https://www.google.ca/search?q=object+spread+operator&rlz=1C5CHFA_enCA778CA778&oq=object+spread+&aqs=chrome.1.69i57j0l5.2734j1j4&sourceid=chrome&ie=UTF-8) until we got a good system in place for the styling.

Reply:
@AbhiPrasad what was this change for?

Reply:
So we pass in a style object into container style. Here I just spread the `FaqStyles.faqListItem` and the `rowBg` into that style object. Eslint complained that the objects were not being passed down properly into containerStyle.

Reply:
Oh interesting, this looks a bit messy.  If we can find a way to make it cleaner, let's do that.  If not, no biggie.